WexHomework4/WexHomework4/Program.cs

```diff
@@ -0,0 +1,109 @@
+﻿
+
+// заполняем массив рандомными числами от -100 до 100
+static void IntializeArray(int[] array, int cofficient)
+{
+    Random rnd = new Random();
+   
+
+    for (int step = 0; step < array.Length; step++)
+    {
+        array[step] = rnd.Next(-100, 100);
+        //Console.WriteLine(array[step]);
+    }
+    
+}
+//находим экстримумы мин или макс
+static int extremumCalculation(int[] arrayExtr,string extremum)
+{    
+    int extr = arrayExtr[0];
+    for (int e = 0; e < arrayExtr.Length; e++)
+    {
+        if (extremum == "min")
+        {
+            if (arrayExtr[e]<extr) extr= arrayExtr[e];
+        }
+        else
+        {
+            if (arrayExtr[e] > extr) extr = arrayExtr[e];
+        }
+    }
+ 
+    return extr;
```

Comment:
you can immediately return min and max like array, for example or tuple

Reply:
I would like to reuse this method to find min or max positions (for my primary and my final arrays) so I need only one value depend on which extremum is set as the parameter. If I change this method I have to change all my logic. What is the profit?